cmd/protoc-gen-go-tableau-loader/helper/helper.go

```diff
@@ -46,22 +45,37 @@ func ParseGoType(file *protogen.File, fd protoreflect.FieldDescriptor) string {
 	case protoreflect.BytesKind:
 		return "[]byte"
 	case protoreflect.MessageKind:
-		fullName := string(fd.Message().FullName())
-		pkg := string(file.Desc.Package())
-		protoName := fullName
-		if strings.HasPrefix(fullName, pkg) {
-			// defined at the same package
-			protoName = fullName[len(pkg)+1:]
+		var mapValueMessage *protogen.Message
+		for _, f := range gen.Files {
+			mapValueMessage = findMessageByDescriptor(f.Messages, fd.Message())
+			if mapValueMessage != nil {
+				break
+			}
```

Comment:
verbose and time-costing, please use ParentFile.

Reply:
done